Resources/config/imagine.xml

```diff
@@ -424,5 +436,10 @@
             <tag name="liip_imagine.filter.post_processor" post_processor="mozjpeg" />
         </service>
 
+        <service id="liip_imagine.filter.post_processor.cwebp" class="Liip\ImagineBundle\Imagine\Filter\PostProcessor\CwebpPostProcessor">
+            <argument>%liip_imagine.cwebp.binary%</argument>
```

Comment:
don't you need to list all parameters here to get them injected to the class?

Reply:
Oops, my bad. I have fixed it.